src/components/SimpleTable.vue

```diff
@@ -12,15 +12,18 @@
           </th>
         </tr>
       </thead>
-      <tbody class="table-body-wrapper">
-        <tr class="row" v-for="entry in filteredData">
+      <tbody class="table-body-wrapper" v-if="data.length > 0">
+        <tr class="row" v-for="entry in filteredData" @click="handleRowClick ? handleRowClick(entry) : () => {}">
```

Comment:
Would this maybe be cleaner if you abstracted it into a method?

Reply:
Yeah for sure.